docs/diagrams/AddItemActivityDiagram.puml

```diff
@@ -0,0 +1,10 @@
+@startuml
+(*)  --> "User Executes AddItemCommand"
+If "Item exists in InventoryBook" then
```

Comment:
If I remember correctly, all the conditions need a [ ... ] right? 
And maybe instead of having true or false like this, it can be replaced as: Item exists in InventoryBook vs Item does not exist in InventoryBook